File: WebAPI/MROS_API/Models/Restaurant.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MROS_API.Models
{
    public class Restaurant
    {
        [Key]
        public int ID { get; set; }
        public string FullName { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Logo { get; set; }
        public string Photo { get; set; }
        public string Information { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public bool Status { get; set; }
        public int OrderCount { get; set; }
        public double TotalGain { get; set; }
    }
}

File: iOS/MROS/MROS/models/Restaurant.swift
//
//  Restaurant.swift
//  MROS
//
//  Created by Tayyip Muslu  on 19.01.2020.
//  Copyright © 2020 MUSLUNET. All rights reserved.
//

import Foundation

struct Restaurant: Codable {
    var id:Int?
    var fullName:String?
    var address:String?
    var phone:String?
    var logo:String?
    var photo:String?
    var information:String?
    var latitude:Double?
    var longitude:Double?
    var status:Bool?
    var orderCount:Int?
    var totalGain:Double?
    
}
